src/hotspot/share/opto/vectornode.cpp

```diff
@@ -976,6 +1018,10 @@ bool VectorNode::is_scalar_unary_op_with_equal_input_and_output_types(int opc) {
   }
 }
 
+VectorNode* VectorReinterpretNode::make(Node* n, const TypeVect* dst_vt, const TypeVect* src_vt) {
+  return new VectorReinterpretNode(n, dst_vt, src_vt);
+}
```

Comment:
This seems like an unnecessary redirection... the arguments and output is the same. Do we need it?